python/cudf/cudf/tests/test_array_ufunc.py

```diff
@@ -23,11 +34,77 @@ def test_ufunc_cudf_series(np_ar_tup, func):
         assert_eq(expect, got.to_array())
 
 
+@pytest.mark.parametrize(
+    "func", [np.bitwise_and, np.bitwise_or, np.bitwise_xor],
+)
+def test_ufunc_cudf_series_bitwise(func):
+    x = np.random.randint(size=100, low=0, high=100)
+    y = np.random.randint(size=100, low=0, high=100)
+
+    s_1, s_2 = cudf.Series(x), cudf.Series(y)
+    expect = func(x, y)
+    got = func(s_1, s_2)
+    if np.isscalar(expect):
```

Comment:
Does this case ever occur if `x` and `y` are both `ndarray`s?

Reply:
nope, we can remove this.